recipe/meta.yaml

```diff
@@ -11,19 +11,20 @@ source:
 
 build:
    number: 0
+
+   {% set VC_VERSION = os.environ.get('VC_VERSION', '15.0') %}
+   msvc_compiler: {{ VC_VERSION }}
    features:
-     - vc9      [win and py27]
-     - vc10     [win and py34]
-     - vc14     [win and py35]
+     - vc{{ VC_VERSION.split('.')[0] }}  # [win]
 
 requirements:
   build:
-    - gmp      [not win]
+    - vc  # [win]
```

Comment:
I'm not sure actually - the vc package doesn't *contain* anything, and I'm not sure it even influences any package resolution. Maybe it does need to be pinned though in order to control the dependencies being installed.